db.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+const bluebird = require("bluebird");
+mongoose.Promise = bluebird;
+mongoose.Promise = global.Promise;
+
```

Comment:
These changes are related to https://www.mongodb.com/blog/post/optimizing-aws-lambda-performance-with-mongodb-atlas-and-nodejs